web/src/test/java/org/cbioportal/genome_nexus/web/mock/HotspotMixin.java

```diff
@@ -0,0 +1,32 @@
+package org.cbioportal.genome_nexus.web.mock;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class HotspotMixin
```

Comment:
This mixin is used only for test purposes, while reading from a json DB dump. There might be a way to bypass this.